index.js

```diff
@@ -69,6 +70,12 @@ const queryClient = new QueryClient( {
           || ( error instanceof TypeError && error.message.match( "Network request failed" ) )
         ) return failureCount < 3;
         handleError( error, { throw: false } );
+        if ( error.status === 401 || error.status === 403 ) {
+          // If we get a 401 or 403, call getJWT
+          // which has a timestamp check if we need to refresh the token
+          const token = await getJWT( );
+          if ( token ) return failureCount < 2;
+        }
```

Comment:
Don't you want to call `handleError` after trying to renew the token?

Reply:
Yes, the idea behind that was to call `handleError` anyway because I added logging to the log file of all handled errors to that function so that we know when 401 and 403 happen. Otherwise, since they are not thrown from here we wouldn't know. All of this logging here I added because I could not reproduce errors with the JWT reliably.